toolchain/check/import.cpp

```diff
@@ -89,53 +89,69 @@ static auto CopyNameFromImportIR(Context& context,
   return import_name_id;
 }
 
+static auto MakeImportedNamespaceLocIdAndInstId(
+    Context& context, SemIR::InstId import_id, SemIR::Namespace& namespace_inst)
```

Comment:
We typically pass small structures by value instead of by reference, for better performance. Also, I don't think this needs to be mutable.

```suggestion
    Context& context, SemIR::InstId import_id, SemIR::Namespace namespace_inst)
```

Reply:
Done.